saleor/graphql/checkout/tests/test_checkout.py

```diff
@@ -1517,6 +1542,26 @@ def test_checkout_lines_add_with_unavailable_variant(
     assert errors[0]["variants"] == [variant_id]
 
 
+def test_checkout_lines_add_with_zero_quantity(
```

Comment:
Missing test for `CheckoutLinesUpdate`  mutation.

Reply:
Which test is missing?
For `CheckoutLinesUpdate` witch zero quantity we already had test - `test_checkout_line_delete_by_zero_quantity`